scripts/usb_scan.py

```diff
@@ -607,19 +607,14 @@ def to_pusb(device):
     return pusb
 
 
-if __name__ == "__main__":
-    args = parse_args()
-    if args.diagnostic:
-        log.logToSyslog(level=logging.DEBUG)
-    else:
-        log.logToSyslog(level=logging.WARNING)
-
-    # get usb info
-    try:
-        devices, interfaces = get_usb_info()
-    except Exception as e:
-        log_exit("Failed to get usb info: {}".format(str(e)))
+def make_pusbs_list(devices, interfaces):
+    """ check the USB devices and interfaces against policy file,
+    and return the pusb list that can be passed through
 
+    :param devices:([UsbDevice]) USB device list we found in host
+    :param interfaces:([UsbInterface]) USB interface list found in host
+    :return:([pusb]) the pusb list that can be passed through
+    """
     # debug info
     log_list(devices)
```

Comment:
Please move these debug info logs to the main body before L648, or remove them completely